leaderboardFeature/src/main/java/com/example/leaderboardfeature/ui/LeaderboardListItem.kt

```diff
@@ -0,0 +1,105 @@
+package com.example.leaderboardfeature.ui
+
+import androidx.compose.foundation.Image
+import androidx.compose.foundation.layout.Box
+import androidx.compose.foundation.layout.Column
+import androidx.compose.foundation.layout.Row
+import androidx.compose.foundation.layout.fillMaxWidth
+import androidx.compose.foundation.layout.padding
+import androidx.compose.foundation.layout.size
+import androidx.compose.foundation.shape.CornerSize
+import androidx.compose.foundation.shape.RoundedCornerShape
+import androidx.compose.material.Card
+import androidx.compose.material.MaterialTheme.typography
+import androidx.compose.material.Text
+import androidx.compose.runtime.Composable
+import androidx.compose.ui.Alignment
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.draw.clip
+import androidx.compose.ui.graphics.Color
+import androidx.compose.ui.layout.ContentScale
+import androidx.compose.ui.res.painterResource
+import androidx.compose.ui.res.stringResource
+import androidx.compose.ui.unit.dp
+import com.example.leaderboardfeature.date.DateType
+import com.example.leaderboardfeature.date.DateType.DAYS
+import com.example.leaderboardfeature.date.DateType.HOURS
+import com.example.leaderboardfeature.date.DateType.MINUTES
+import com.example.leaderboardfeature.date.DateType.MONTHS
+import com.example.leaderboardfeature.date.DateType.YEARS
+import com.example.leaderboardfeature.R.string
```

Comment:
It's better not to import R.string and specify the whole resource id like: `R.string.resource` instead of `string.resource` as it looks confusing